src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+
+export const SignupView = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [birthday, setBirthday] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const data = {
+      Username: username,
+      Password: password,
+      Email: email,
+      Birthday: birthday
+    };
+
+    fetch("https://myflix-ghibli-7c8d5913b80b.herokuapp.com/users", {
+      method: "POST",
+      body: JSON.stringify(data),
+      headers: {
+        "Content-Type": "application/json"
+      }
+    }).then((response) => {
+      if (response.ok) {
+        alert("Signup successful!");
+        window.location.reload();
```

Comment:
In react apps, its generally a good practice to navigate instead of reloading, as if you were to add any context or state values to this part of the app later the reload would reset them. It is fine here right now, but consider using `useNavigate`